<ui version="4.0" >
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mumble Server Connect</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="qtwTab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Custom Servers</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QListView" name="qlwServers" >
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>&amp;Label</string>
             </property>
             <property name="buddy" >
              <cstring>qleName</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>A&amp;ddress</string>
             </property>
             <property name="buddy" >
              <cstring>qleServer</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>&amp;Port</string>
             </property>
             <property name="buddy" >
              <cstring>qlePort</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>&amp;Username</string>
             </property>
             <property name="buddy" >
              <cstring>qleUsername</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>&amp;Password</string>
             </property>
             <property name="buddy" >
              <cstring>qlePassword</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="qleName" />
           </item>
           <item row="1" column="1" >
            <widget class="QLineEdit" name="qleServer" />
           </item>
           <item row="2" column="1" >
            <widget class="QLineEdit" name="qlePort" />
           </item>
           <item row="3" column="1" >
            <widget class="QLineEdit" name="qleUsername" />
           </item>
           <item row="4" column="1" >
            <widget class="QLineEdit" name="qlePassword" >
             <property name="echoMode" >
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="qpbConnect" >
           <property name="text" >
            <string>&amp;Connect</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qpbCancel" >
           <property name="text" >
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qpbAdd" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qpbRemove" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Server &amp;Browser</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTreeWidget" name="qtwServers" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Label</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>URL</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="qpbBrowserConnect" >
           <property name="text" >
            <string>&amp;Connect</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qpbBrowserCancel" >
           <property name="text" >
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qpbCopy" >
           <property name="text" >
            <string>C&amp;opy to custom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qpbURL" >
           <property name="text" >
            <string>&amp;View Webpage</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qpbConnect</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>21</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>444</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qlwServers</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>109</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel" >
     <x>133</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qpbBrowserConnect</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>119</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qpbBrowserCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
  </connection>
  <connection>
   <sender>qtwServers</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>158</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel" >
     <x>185</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
